packages/react-mongo/react-mongo.ts

```diff
@@ -1,62 +1,48 @@
 import { Meteor } from 'meteor/meteor'
 import { Mongo } from 'meteor/mongo'
 import { Tracker } from 'meteor/tracker'
-import { useEffect, useMemo, useReducer, useRef, DependencyList } from 'react'
+import { useEffect, useMemo, useReducer, useState, DependencyList } from 'react'
 
 const fur = (x: number): number => x + 1
 const useForceUpdate = () => useReducer(fur, 0)[1]
 
-const useSubscriptionClient = (factory: () => Meteor.SubscriptionHandle | void, deps: DependencyList= []) => {
-  const forceUpdate = useForceUpdate()
-  const { current: refs } = useRef<{
-    handle?: Meteor.SubscriptionHandle,
-    updateOnReady: boolean
-  }>({
-    handle: {
-      stop () {
-        refs.handle?.stop()
-      },
-      ready () {
-        refs.updateOnReady = true
-        return refs.handle?.ready()
-      }
-    },
-    updateOnReady: false
-  })
+const useSubscriptionClient = (
+  name: string | false,
+  args: any[]
```

Comment:
if possible I'd think it would be a good idea to mimick Meteor.subscribe()'s signature : 
useSubscription(name, ...args) 
rather than 
useSubscription(name, single_array_of_args) ?

Reply:
Thanks, I did that on `useSubscription`, which is the user-facing API. If I did that here as well, I could no longer destructure to use it as `useEffect` deps.